### ISSUE TITLE
Unable to delete a Tenant with a name longer than 48 chars

### ISSUE THREAD

**Issue by erjac77** [F5Networks/f5-appsvcs-extension#209]
### Environment
 * Application Services Version: `3.13.1`
 * BIG-IP Version: `BIG-IP 13.1.1.4 Build 0.0.4 Point Release 4`

### Summary
A successfully deployed declaration cannot be deleted when it contains a Tenant name longer than 48 characters. According to the documentation, the maximum length of the property name should be 64 characters.

https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/refguide/understanding-the-json-schema.html

### Steps To Reproduce
1. Submit the following declaration with a Tenant name of 50 chars:
```json
{
  "class": "AS3",
  "action": "deploy",
  "persist": true,
  "declaration": {
    "class": "ADC",
    "schemaVersion": "3.0.0",
    "label": "Sample HTTP application",
    "remark": "Simple HTTP application with round robin pool",
    "sample01234567890123456789012345678901234567890123": {
      "class": "Tenant",
      "defaultRouteDomain": 0,
      "Application_1": {
        "class": "Application",
        "template": "http",
        "serviceMain": {
          "class": "Service_HTTP",
          "virtualAddresses": ["10.11.11.111"],
          "pool": "web_pool"
        },
        "web_pool": {
          "class": "Pool",
          "monitors": ["http"],
          "members": [
            {
              "servicePort": 80,
              "serverAddresses": ["10.11.11.101", "10.11.11.102"]
            }
          ]
        }
      }
    }
  }
}
```

2. Observe the following **successful** response:
```json
        {
            "message": "success",
            "lineCount": 25,
            "code": 200,
            "host": "localhost",
            "tenant": "sample01234567890123456789012345678901234567890123",
            "runTime": 1597
        }
```

3. Use the DELETE method to remove the previously posted declaration:
```
DELETE https://x.x.x.x/mgmt/shared/appsvcs/declare/sample01234567890123456789012345678901234567890123
```

2. Observe the following **error** response:
```json
{
    "code": 400,
    "message": "invalid Tenant name \"sample01234567890123456789012345678901234567890123\""
}
```

### Expected Behavior
The Tenant should be successfully deleted.

### Actual Behavior
An error message mentioning an invalid Tenant name is received and the Tenant is not deleted.

**Comment by dstokesf5** [comment]
This is tracked in our internal product backlog as AUTOTOOL-868.

**Comment by dstokesf5** [comment]
There were some fixes to tenant name validation on DELETE and PATCH requests, and I am no longer seeing this issue with 3.19.0.

Closing this issue. If you are still seeing a problem, please re-open this issue or create a new one.

**Comment by erjac77** [comment]
Thx @dstokesf5.

However, I would also expect this bug fix to be rolled into the Stability (LTS) releases (eg 3.13.x). Can you confirm that?

**Comment by dstokesf5** [comment]
There are no plans to back-port the fix to 3.13 at this time. We are preparing to make 3.19 LTS, and the fix will be available there.

**Comment by erjac77** [comment]
Ok great then if 3.19.0 is going to be LTS. Thanks again!
